packages/block-editor/src/hooks/block-bindings.js

```diff
@@ -174,31 +200,41 @@ function EditableBlockBindingsPanelItems( {
 								[ attribute ]: undefined,
 							} );
 						} }
+						onSelect={ () => setOuterModalOpen( true ) }
 					>
-						<Menu
-							placement={
-								isMobile ? 'bottom-start' : 'left-start'
-							}
-							gutter={ isMobile ? 8 : 36 }
-							trigger={
-								<Item>
-									<BlockBindingsAttribute
-										attribute={ attribute }
-										binding={ binding }
-										fieldsList={ fieldsList }
-									/>
-								</Item>
-							}
+						<Item
+							key={ attribute }
+							onClick={ () => {
+								setOuterModalOpen( true );
+							} }
 						>
-							<BlockBindingsPanelDropdown
-								fieldsList={ fieldsList }
+							<BlockBindingsAttribute
 								attribute={ attribute }
 								binding={ binding }
+								fieldsList={ fieldsList }
 							/>
-						</Menu>
+						</Item>
 					</ToolsPanelItem>
 				);
 			} ) }
+			{ isOuterModalOpen && (
+				<Modal
+					onRequestClose={ () => setOuterModalOpen( false ) }
+					__experimentalHideHeader
+					className="block-editor-bindings__modal"
+				>
+					<DataViews
+						getItemId={ ( item ) => item.label }
```

Comment:
I [see](https://github.com/WordPress/gutenberg/pull/67012/files#diff-bff98e322c4269c50aa13e4b251beb9d53d2d0c3d4066406f735d2d49bed83a1R102) the `dataSet` is an array of objects that have the following keys: `id`, `label`, `value`. Is the `id` unique among the items? It seems so. If my understanding is correct, `getItemId` could be removed here.

Reply:
Done